scene2d/src/test/kotlin/ktx/scene2d/factoryTest.kt

```diff
@@ -142,7 +151,15 @@ class NoInitBlockActorFactoriesTest : ApplicationTest() {
   fun `should create Image with drawable`() = test(
     widget = { image(VisUI.getSkin().getDrawable("button")) },
     validate = {
-      Assert.assertSame(VisUI.getSkin().getDrawable("button"), it.drawable)
+      assertSame(VisUI.getSkin().getDrawable("button"), it.drawable)
+    },
+  )
+
+  @Test
+  fun `should create Image with null drawable`() = test(
```

Comment:
I'd appreciate it if you added another test for `image()` without a parameter just to see that it compiles and creates an image without a drawable.